packages/react-html/README.md

```diff
@@ -226,6 +226,10 @@ This component accepts a string child, which will be used to set the title of th
 
 Renders a `<link />` tag with the necessary props to specify a favicon. Accepts a `source` property that should be the image source for the favicon.
 
+### `<Preconnect />`
+
+Renders a `<link />` tag with the necessary props to fully preconnect to the host specified by the `source` prop.
```

Comment:
Just a thought: I'm on the fence if the word `necessary` is too vague and we should just say what it does. I understand you are trying to shield the implementation details to the consumer, which is why I'm on the fence.